packages/front-generator/src/generators/react-typescript/app/template/src/app/App.tsx

```diff
@@ -57,13 +59,14 @@ class AppComponent extends React.Component<
             style={{ background: "#fff" }}
           >
             <Menu mode="inline" style={{ height: "100%", borderRight: 0 }}>
-              <Menu.Item key="1">
+              <Menu.Item key={`menu-item-${menuIdx}`}>
```

Comment:
I think there is no point in using any suffix in `key`.
Also consider this
https://reactjs.org/docs/lists-and-keys.html#keys
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318